en/micro-integrator/docs/use-cases/examples/data_integration/data-input-validator.md

```diff
@@ -75,40 +75,41 @@ Create the artifacts:
 4. [Create the data service](../../../../develop/creating-artifacts/data-services/creating-data-services) with the configurations given above.
 5. [Deploy the artifacts](../../../../develop/deploy-and-run) in your Micro Integrator. 
 
-The **Deployed Services** window allows you to manage data services. You
-can try the data service you created by using the TryIt tool in this
-screen, which is in your product by default.
+Let's send a request with invalid and valid data to the data service:
 
-1.  Invoke the **addEmployeeOp** operation:
-```xml
-<addEmployeeOp xmlns="http://ws.wso2.org/dataservice">
-     <!--Exactly 1 occurrence-->
-     <xs:EmployeeNumber xmlns:xs="http://ws.wso2.org/dataservice">6001</xs:EmployeeNumber>
-     <!--Exactly 1 occurrence-->
-     <xs:FirstName xmlns:xs="http://ws.wso2.org/dataservice">AB</xs:FirstName>
-     <!--Exactly 1 occurrence-->
-     <xs:LastName xmlns:xs="http://ws.wso2.org/dataservice">Nick</xs:LastName>
-     <!--Exactly 1 occurrence-->
-     <xs:Email xmlns:xs="http://ws.wso2.org/dataservice">test@test.com</xs:Email>
-     <!--Exactly 1 occurrence-->
-     <xs:Salary xmlns:xs="http://ws.wso2.org/dataservice">1500</xs:Salary>
-</addEmployeeOp>
-```
-A validation error is thrown as the response because the addEmployeeOp operation has failed. This is because the FirstName only has 2 characters.
+1. Download and Install [SoapUI](https://www.soapui.org/downloads/soapui.html) to run this SOAP service.
+2. Create a new SOAP project in SoapUI by using the following wsdl file:
+   ```bash
+   http://localhost:8290/services/input_validator_sample?wsdl
+   ```
 
-2. Now, change the FirstName in the request as
-```xml
-<addEmployeeOp xmlns="http://ws.wso2.org/dataservice">
-     <!--Exactly 1 occurrence-->
-     <xs:EmployeeNumber xmlns:xs="http://ws.wso2.org/dataservice">6001</xs:EmployeeNumber>
-     <!--Exactly 1 occurrence-->
-     <xs:FirstName xmlns:xs="http://ws.wso2.org/dataservice">ABC</xs:FirstName>
-     <!--Exactly 1 occurrence-->
-     <xs:LastName xmlns:xs="http://ws.wso2.org/dataservice">Nick</xs:LastName>
-     <!--Exactly 1 occurrence-->
-     <xs:Email xmlns:xs="http://ws.wso2.org/dataservice">test@test.com</xs:Email>
-     <!--Exactly 1 occurrence-->
-     <xs:Salary xmlns:xs="http://ws.wso2.org/dataservice">1500</xs:Salary>
-</addEmployeeOp>
-```
-The employee details are added to the database table.  
+3. Update the **addEmployeeOp** operation (under **input_validator_sample.SOAP12Binding**) with the request body as shown below:
+
+   !!! Tip
```

Comment:
@rosensilva - Is this tip applicable to this example?

Reply:
Fixed